github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRLabelExistsEvent/config.groovy

```diff
@@ -2,7 +2,7 @@ package org.jenkinsci.plugins.github.pullrequest.events.impl.GitHubPRLabelExists
 
 def f = namespace(lib.FormTagLib);
 
-f.entry(field: "skip", title: "Skip PR?") {
+f.entry(field: "skip", title: "Skip pull requests with label(s)?") {
```

Comment:
with existed label would be critical, because it has other events that works in comparison mode i.e. label existed, but was removed and vice versa.